File: crianca.php
<?php
    $servername = "localhost";
    $username = "root";
    $password = "";
    $dbname = "pdm";
    $table = "crianca";

    //$action = $_GET['acao'];

    // Create connection
    $conn = new mysqli($servername, $username, $password, $dbname);
    // Check connection
    if ($conn->connect_error) {
        die("Connection failed: " . $conn->connect_error);
        return;
    } else {
        echo "ok";

    }

    $nome = $_POST["nome"];
    $dataNasc = $_POST["dataNasc"];
    $sexo = $_POST["sexo"];
    $descricao = $_POST["descricao"];

    $query = "insert into crianca (id, nome, dataNasc, sexo, descricao) values (null, '$nome', '$dataNasc', '$sexo', '$descricao')";
    $result = mysqli_query($conn, $query);

    if($result){
        echo "ok";
    } else {
        echo "error";
    }

?>

File: lib/Crianca.dart
class Crianca {
  String _id;
  String _nome;
  String _dataNasc;
  String _sexo;
  String _descricao;

  Crianca(this._id, this._nome, this._dataNasc, this._sexo, this._descricao);

  String get descricao => _descricao;

  set descricao(String value) {
    _descricao = value;
  }

  String get sexo => _sexo;

  set sexo(String value) {
    _sexo = value;
  }

  String get dataNasc => _dataNasc;

  set dataNasc(String value) {
    _dataNasc = value;
  }

  String get nome => _nome;

  set nome(String value) {
    _nome = value;
  }

  String get id => _id;

  set id(String value) {
    _id = value;
  }

}
